addons/website_slides/static/src/js/slides_course_page.js

```diff
@@ -20,6 +20,23 @@ export const SlideCoursePage = publicWidget.Widget.extend({
         'slide_mark_completed': '_onSlideMarkCompleted',
     },
 
+    /**
+     * @override
+     */
+    start: function () {
+        // TODO: we need to clean this code and make the changes in the view in master
+        return this._super.apply(this, arguments).then(()=>{
+            const $completed = $('.o_wslides_channel_completion_completed');
+            const $progressbar = $('.o_wslides_channel_completion_progressbar');
+            if($progressbar.hasClass('d-none')){
+                $progressbar.removeClass('d-none').addClass('d-flex').addClass('hidden-progressbar-completed-tag');
+            }
+            if($completed.hasClass('d-none')){
+                $completed.removeClass('d-none').addClass('hidden-progressbar-completed-tag');
+            }
```

Comment:
Hi, I haven't checked about the test, but it seems legit to have a call to `super`.

If you do want to return something, and it should be the result of `super` (still makes sense), couldn't you simply use `return this._super.apply(this, arguments):` at the end of the existing patch of the FW port?
And if the parent should be called first, we prefer to do something like this:

```js
const result = await this._super.apply(this, arguments):
... // other operations, without `.then`
return result;
```

Reply:
I fixed it , thank you for the suggestions.